livedevice/models.py

```diff
@@ -188,8 +189,22 @@ def set_webhook_callback(self):
         webhook_api_key = self.webhookauth.api_key
         headers = {'Authorization': "Bearer %s" % webhook_api_key}
         session.set_callback(webhook_url, headers=headers)
+        self.request_initial_resource_values()
         self.set_presubscriptions()
 
+    def request_initial_resource_values(self):
+        '''Request values for resource paths to obtain initial values.'''
+        session = self.get_session()
+        endpoints = session.get_endpoints()
+        for e in endpoints:
+            endpoint_id = e['name']
+            for p in settings.RESOURCE_PATHS_FOR_INIT:
```

Comment:
Is there a commit that adds this setting?

Reply:
That was added: https://github.com/ARMmbed/ref-wem-webapp/commit/b1dd4f994ae78abbeeb90371ddd614221b856cc6